Backend/lib/message-service/message-service.cabal

```diff
@@ -0,0 +1,125 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.35.2.
+--
+-- see: https://github.com/sol/hpack
+
+name:           message-service
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/nammayatri/nammayatri#readme>
+homepage:       https://github.com/“nammayatri/nammayatri”#readme
+bug-reports:    https://github.com/“nammayatri/nammayatri”/issues
+author:         “nammayatri”
+maintainer:     “nammayatri”
+copyright:      “2023 Juspay Technologies Private Limited”
+license:        AllRightsReserved
+build-type:     Simple
+
+source-repository head
+  type: git
+  location: https://github.com/“nammayatri/nammayatri”
+
+library
+  exposed-modules:
+      Lib.API.Dashboard.Message
+      Lib.API.UI.Message
+      Lib.Domain.Action.Dashboard.Message
+      Lib.Domain.Action.UI.Message
+      Lib.Domain.Types.MediaFile
+      Lib.Domain.Types.Message.Message
+      Lib.Domain.Types.Message.MessageReport
+      Lib.Domain.Types.Message.MessageTranslation
+      Lib.Storage.Queries.MediaFile
+      Lib.Storage.Queries.Message.Message
```

Comment:
`.Message.Message` Why some module has submodule with the same name? It doens't make sense as well

Reply:
I just replicated the structure as it was in BPP side.

Reply:
> I just replicated the structure as it was in BPP side.

Yes, i got this much, but it doesn't mean that this thing is correct

Reply:
This is also not resolved. I also added the same comment in later files.